### PR TITLE
Merge 3.8.6 beta to master

### PR DESCRIPTION
**Pull request by enishant**

### CODE REVIEW

File: .semgrepignore [hunk 1]
```diff
@@ -1 +0,0 @@
-.github/workflows/
```
Comment (ramth05) [inline .semgrepignore]: @enishant , please don't delete this one.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,89 +0,0 @@
-
```
Comment (ramth05) [inline CHANGELOG.md]: @enishant  , this one also needs to be maintained.

File: etc/graphql/di.xml [hunk 3]
```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0"?>
```
Comment (ramth05) [inline etc/graphql/di.xml]: @enishant , why we are deleting this one?

File: .semgrepignore (hunk 1 above)
Comment (enishant) [reply .semgrepignore]: Included file(s) which was get deleted while git merge ours.

File: CHANGELOG.md (hunk 2 above)
Comment (enishant) [reply CHANGELOG.md]: Included file(s) which was get deleted while git merge ours and updated change log.

File: etc/graphql/di.xml (hunk 3 above)
Comment (enishant) [reply etc/graphql/di.xml]: Included file(s) which was get deleted while git merge ours.

**Review by ramth05 — APPROVED** [review]
LGTM

**Pull request merged**